[docs] Add alt text on images (related to #4036) (#4038)
microsoft/LightGBM: docs/GPU-Windows.rst

.. image:: ./_static/images/screenshot-system.png
   :align: center
   :target: ./_static/images/screenshot-system.png

>    :alt: A screenshot of the System option under System and Security of the Control Panel

Reviewer: ```suggestion
   :alt: A screenshot of the System option under System and Security of the Control Panel.
```

Then, go to ``Advanced`` > ``Environment Variables...``:

.. image:: ./_static/images/screenshot-advanced-system-settings.png
   :align: center
   :target: ./_static/images/screenshot-advanced-system-settings.png

>    :alt: A screenshot of the System Properties window

Reviewer: ```suggestion
   :alt: A screenshot of the System Properties window.
```

Under ``System variables``, the variable ``Path``:

.. image:: ./_static/images/screenshot-environment-variables.png
   :align: center
   :target: ./_static/images/screenshot-environment-variables.png

>    :alt: A screenshot of the Environment variables window with variable path selected under the system variables

Reviewer: ```suggestion
   :alt: A screenshot of the Environment variables window with variable path selected under the system variables.
```

.. image:: ./_static/images/screenshot-mingw-installation.png
   :align: center
   :target: ./_static/images/screenshot-mingw-installation.png

>    :alt: A screenshot of the Min G W installation setup settings window

Reviewer: ```suggestion
   :alt: A screenshot of the Min G W installation setup settings window.
```

>    :alt: A screenshot of the Min G. W. installation setup settings window

Reviewer 2: ```suggestion
   :alt: A screenshot of the Min G W installation setup settings window
```

.. image:: ./_static/images/screenshot-r-mingw-used.png
   :align: center
   :target: ./_static/images/screenshot-r-mingw-used.png

>    :alt: A screenshot of the administrator command prompt where G C C version is being checked 

Reviewer: ```suggestion
   :alt: A screenshot of the administrator command prompt where G C C version is being checked.
```

>    :alt: A screenshot of the administrator command prompt where G. C. C. version is being checked 

Reviewer 2: ```suggestion
   :alt: A screenshot of the administrator command prompt where G C C version is being checked 
```

.. image:: ./_static/images/screenshot-boost-compiled.png
   :align: center
   :target: ./_static/images/screenshot-boost-compiled.png

>    :alt: A screenshot of the command prompt that ends with text that reads - updated 14621 targets

Reviewer: ```suggestion
   :alt: A screenshot of the command prompt that ends with text that reads - updated 14621 targets.
```

.. image:: ./_static/images/screenshot-git-for-windows.png
   :align: center
   :target: ./_static/images/screenshot-git-for-windows.png

>    :alt: A screenshot of the website to download git that shows various versions of git compatible with 32 bit and 64 bit Windows separately

Reviewer: ```suggestion
   :alt: A screenshot of the website to download git that shows various versions of git compatible with 32 bit and 64 bit Windows separately.
```

.. image:: ./_static/images/screenshot-downloading-cmake.png
   :align: center
   :target: ./_static/images/screenshot-downloading-cmake.png

>    :alt: A screenshot of the binary distributions of c-make for downloading on Windows 64 bit

Reviewer 2: ```suggestion
   :alt: A screenshot of the binary distributions of C Make for downloading on 64 bit Windows
```

I think the way you separated this out into `C` and `Make` will be better understood by screen readers.

>    :alt: A screenshot of the binary distributions of C Make for downloading on 64 bit Windows

Reviewer: ```suggestion
   :alt: A screenshot of the binary distributions of C Make for downloading on 64 bit Windows.
```

   .. image:: ./_static/images/screenshot-create-directory.png
      :align: center
      :target: ./_static/images/screenshot-create-directory.png

>       :alt: A screenshot with a pop-up window that reads - Build directory does not exist, should I recreate it? 

Reviewer: ```suggestion
      :alt: A screenshot with a pop-up window that reads - Build directory does not exist, should I create it?
```

   .. image:: ./_static/images/screenshot-mingw-makefiles-to-use.png
      :align: center
      :target: ./_static/images/screenshot-mingw-makefiles-to-use.png

>       :alt: A screenshot that asks to sepcify the generator for the project which should be selected as Min G W makefiles and selected as the use default native compilers option

Reviewer: ```suggestion
      :alt: A screenshot that asks to specify the generator for the project which should be selected as Min G W makefiles and selected as the use default native compilers option.
```

   .. image:: ./_static/images/screenshot-configured-lightgbm.png
      :align: center
      :target: ./_static/images/screenshot-configured-lightgbm.png

>       :alt: A screenshot of the C Make window after clicking on the configure button

Reviewer: ```suggestion
      :alt: A screenshot of the C Make window after clicking on the configure button.
```

.. image:: ./_static/images/screenshot-lightgbm-with-gpu-support-compiled.png
   :align: center
   :target: ./_static/images/screenshot-lightgbm-with-gpu-support-compiled.png

>       :alt: A screenshot of the gitbash window with Light G B M successfully installed

Reviewer 2: ```suggestion
      :alt: A screenshot of the git bash window with Light G B M successfully installed
```

>       :alt: A screenshot of the git bash window with Light G B M successfully installed

Reviewer: ```suggestion
   :alt: A screenshot of the git bash window with Light G B M successfully installed.
```

.. image:: ./_static/images/screenshot-lightgbm-in-cli-with-gpu.png
   :align: center
   :target: ./_static/images/screenshot-lightgbm-in-cli-with-gpu.png

>       :alt: A screenshot of the command prompt where a binary classification model is being trained using Light G B M

Reviewer: ```suggestion
   :alt: A screenshot of the command prompt where a binary classification model is being trained using Light G B M.
```

.. image:: ./_static/images/screenshot-segmentation-fault.png
   :align: center
   :target: ./_static/images/screenshot-segmentation-fault.png

>    :alt: A screenshot of the command prompt where a segmentation fault has occured while using Light B G M

Reviewer 2: ```suggestion
   :alt: A screenshot of the command prompt where a segmentation fault has occurred while using Light G B M
```

I make this mistake a lot in this project haha

Author: Dang T_T

>    :alt: A screenshot of the command prompt where a segmentation fault has occurred while using Light G B M

Reviewer: ```suggestion
   :alt: A screenshot of the command prompt where a segmentation fault has occurred while using Light G B M.
```

.. image:: ./_static/images/screenshot-files-to-remove.png
   :align: center
   :target: ./_static/images/screenshot-files-to-remove.png

>    :alt: A screenshot of the Light G B M folder with 1 folder and 3 files selected to be removed

Reviewer: ```suggestion
   :alt: A screenshot of the Light G B M folder with 1 folder and 3 files selected to be removed.
```

Once you removed the file, go into CMake, and follow the usual steps.
Before clicking "Generate", click on "Add Entry":

.. image:: ./_static/images/screenshot-added-manual-entry-in-cmake.png
   :align: center
   :target: ./_static/images/screenshot-added-manual-entry-in-cmake.png

>    :alt: A screenshot of the Cache Entry popup where the name is set to C Make_Build_Type in all caps, the type is set to STRING in all caps and the value is set to Debug

Reviewer: ```suggestion
   :alt: A screenshot of the Cache Entry popup where the name is set to CMAKE_BUILD_TYPE in all caps, the type is set to STRING in all caps and the value is set to Debug.
```

In addition, click on Configure and Generate:

.. image:: ./_static/images/screenshot-configured-and-generated-cmake.png
   :align: center
   :target: ./_static/images/screenshot-configured-and-generated-cmake.png

>    :alt: A screenshot of the C Make window after clicking on configure and generate

Reviewer: ```suggestion
   :alt: A screenshot of the C Make window after clicking on configure and generate.
```

.. image:: ./_static/images/screenshot-debug-run.png
   :align: center
   :target: ./_static/images/screenshot-debug-run.png

>    :alt: A screenshot of the command prompt after the command above is run

Reviewer: ```suggestion
   :alt: A screenshot of the command prompt after the command above is run.
```
